src/hardware/ipx.cpp

```diff
@@ -1045,10 +1045,18 @@ class IPXNET final : public Program {
 				WriteOut("Sending broadcast ping:\n\n");
 				pingSend();
 				ticks = GetTicks();
-				while((GetTicks() - ticks) < 1500) {
+				while ((GetTicksSince(ticks)) < 1500) {
```

Comment:
Loving the `GetTicksSince()` call.. reads a lot easier on the eyes, as I pass over all this code.